content/en/docs/tasks/configure-pod-container/pull-image-private-registry.md

```diff
@@ -77,28 +77,28 @@ The output is similar to this:
 
     apiVersion: v1
     data:
-      .dockerconfigjson: eyJodHRwczovL2luZGV4L ... J0QUl6RTIifX0=
```

Comment:
also? may be i little knowledge about it

Reply:
When you run the `kubectl create secret docker-registry` command, the output no longer uses `.dockerconfigjson` as the key, it now uses `.dockercfg` as the key.